src/demo/html/demo.html

```diff
@@ -1,27 +1,27 @@
+<!DOCTYPE html>
 <html>
-<style>
-
-.theFrame{
-   height: 90%;
-   width: 100%;
-   /*border-style: none;*/
-}
-
-.toolbar {
-   width: 100%;
-   height 30px;
-   background-color: grey;
-}
-
-</style>
-
+<head>
+  <title>Photon Demo Page</title>
+  <style>
+  .theframe{
```

Comment:
While we're tidying ¯\_(ツ)_/¯ 
```suggestion
  .theframe {
```